node/node.go

```diff
@@ -123,15 +124,17 @@ type Node struct {
 	State      State      // State of the Node
 	stateMutex sync.Mutex // mutex for change node state
 
-	TxPool *core.TxPool
-	Worker *worker.Worker
+	TxPool       *core.TxPool
+	Worker       *worker.Worker
+	BeaconWorker *worker.Worker // worker for beacon chain
```

Comment:
I believe we don't need it

Reply:
Let me check if we can use one worker for multiple chains

Reply:
I think it's good to use two workers separately.